ansible/playbooks/bootstrap.yml

```
@@ -10,3 +10,9 @@
   become: True
   roles:
     - user-creation
+
+- name: Add user accounts to seal.purdue.wtf
```

```suggestion
- name: Add user accounts to tsel.purdue.wtf
```

You could also change the previous block to this instead:
```yaml
- name: Add user accounts
  hosts: tsel
  become: True
  roles:
    - user-creation
```

---

Ah I like this more!